js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+adminThemeHelp=Wählen Sie ein Thema für die Administration.
+emailTheme=Thema für E-Mails
+emailThemeHelp=Wählen Sie ein Thema für E-Mails, die vom Server gesendet werden.
+keys=Schlüssel
+keysList=Schlüsselliste
+providers=Anbieter hinzufügen
+keysFilter.ACTIVE=Aktive Schlüssel
+keysFilter.PASSIVE=Passive Schlüssel
+keysFilter.DISABLED=Deaktivierte Schlüssel
+algorithm=Algorithmus
+use=Verwendung
+provider=Anbieter
+validTo=Gültig bis
+publicKeys=Öffentlicher Schlüssel
+eventListeners=Ereignis-Listener
+eventListenersHelpTextHelp=Ereignis-Listener die Ereignisse für den Realm behandeln.
+userEventsSettings=Benutzerereignisse
+adminEventsSettings=Administrationsereignisse
+saveEvents=Ereignisse speichern
+save-user-eventsHelp=Wenn diese Option aktiviert ist, werden die Benutzerereignisse in der Datenbank gespeichert, wodurch die Ereignisse für die Administrations- und Benutzerkontokonsolen verfügbar werden.
+revert=Zurücksetzen
+clearUserEvents=Benutzerereignisse löschen
+user-clearEventsHelp=Löscht alle Benutzerereignisse in der Datenbank.
+refresh=Aktualisieren
+addSavedTypes=Gespeicherte Ergeignis-Typen hinzufügen
+searchEventType=Gespeicherte Ereignisart suchen
+eventType=Art des gespeicherten Ereignisses
+expiration=Ablaufdatum
+expirationHelp=Legt das Ablaufdatum für Ereignisse fest. Abgelaufene Ereignisse werden in regelmäßigen Abständen aus der Datenbank gelöscht.
+on=An
+off=Aus
+clearAdminEvents=Admin-Ereignisse löschen
+admin-clearEventsHelp=Löscht alle Verwaltungsereignisse in der Datenbank.
+includeRepresentation=Repräsentation speichern
+includeRepresentationHelp=JSON-Darstellung für Erstellungs- und Aktualisierungsanfragen speichern.
+internationalization=Internationalisierung
+internationalizationHelp=Wenn diese Option aktiviert ist, können Sie wählen, welche Gebietsschemata Sie für diesen Bereich unterstützen und welches Gebietsschema standardmäßig verwendet werden soll.
+locales=Gebietsschema
+defaultLocale=Standard Gebietsschema
+realmOverrides=Realm Überschreibungen
+realmOverridesDescription=Mit Realm-Überschreibungen können Sie Übersetzungen angeben, die für den gesamten Realm gelten sollen. Diese Übersetzungen haben Vorrang vor allen Übersetzungen, die in einem Thema angegeben sind.
+searchForTranslation=Nach Übersetzung suchen
+addTranslation=Übersetzung hinzufügen
+noTranslations=Keine eigenen Übersetzungen vorhanden
+noTranslationsInstructions=Fügen Sie eigene Übersetzungen hinzu.
+effectiveMessageBundles=Effektive Message-Bundles
+effectiveMessageBundlesHelp=Sie können nach effektiven Message-Bundles auf der Grundlage von Thema, Funktionen, Sprache und freiem Text suchen.
+effectiveMessageBundlesDescription=Ein effektives Message-Bundle ist der Satz von Übersetzungen für eine bestimmte Sprache, ein bestimmtes Thema und einen bestimmten Thementyp. Es berücksichtigt auch alle Überschreibungen des Bereichs, die Vorrang haben.
+searchForEffectiveMessageBundles=Nach Message-Bundle suchen
+theme=Thema
```

Comment:
As @SamTV12345 said, I would probably use "theme" everywhere, this seems to be somewhat common in German (e.g. see https://wordpress.com/de/themes).

Reply:
Did so.